### PR TITLE
Add a scrub_field processor similar to honeytail's

### LINKED ISSUE #60: scrub_field processor
The honeytail docs talk about both `--scrub_field` and `--drop_field` flags.

The examples and docs for honeycomb-kubernetes-agent mention that `drop_field` is a supported processor, but don't mention `scrub_field`.

Having tested locally (I have a CI script that runs the agent with my config and `--validate`), I get: `Error setting up processor: Unknown processor type scrub_field`.

Can we have `scrub_field` in the k8s agent, or a clear statement in docs that it's not supported?

### PR DESCRIPTION
**Pull request by ismith**
Fixes #60

### CODE REVIEW

**Comment by ismith** [comment]
Full disclosure: I have not yet tested this in-cluster, just added & ran the automated tests. I don't have a test cluster to work with at the moment.

File: docs/configuration-reference.md [hunk 1]
```diff
@@ -138,6 +138,16 @@ key | value | description
...
+### scrub_field
+
+The `scrub_field` processor will replace the specified field with a one-way hash before sending them to Honeycomb. This is useful for sanitizing sensitive information from events, while still allowing for inclusion and frequency analysis. May be specified multiple times.
+
+**Options:**
+
+key | value | description
+:--|:--|:--
+field | string | The name of the field to scrub.
+
```
Comment (puckpuck) [inline docs/configuration-reference.md:150]: Can you add an example configuration snippet here as well? Similar to what we have for the `addtional_fields` processor. This is part of a larger effort where we need to have clearer docs on how to use these options.

Comment (ismith) [reply docs/configuration-reference.md:150]: How about bd3578e ?

Comment (ismith) [reply docs/configuration-reference.md:150]: How about bd3578e?

**Pull request merged**